packages/sdk/src/sdk/services/Storage/Storage.ts

```diff
@@ -127,31 +128,29 @@ export class Storage implements StorageService {
       file.name ?? 'blob'
     )
 
-    const signatureData = {
-      timestamp: Date.now()
-    }
-    const sigJson = JSON.stringify(sortObjectKeys(signatureData))
-    const signature = await auth.hashAndSign(sigJson)
-    const signatureEnvelope = {
-      data: sigJson,
-      signature
-    }
-
     // Using axios for now because it supports upload progress,
     // and Node doesn't support XmlHttpRequest
     let response: AxiosResponse<any> | null = null
     const request: AxiosRequestConfig = {
       method: 'post',
       maxContentLength: Infinity,
       data: formData,
-      params: { signature: JSON.stringify(signatureEnvelope) },
-      headers: formData.getBoundary
-        ? {
-            'Content-Type': `multipart/form-data; boundary=${formData.getBoundary()}`
-          }
-        : undefined,
-      onUploadProgress: (progressEvent) =>
-        onProgress?.(progressEvent.loaded, progressEvent.total)
+      headers: {
+        'X-User-Wallet-Addr': await auth.getAddress(),
```

Comment:
I thought mediorum got updated to recover the signer from auth headers as an alternate option?

Also, does it matter if the signer here is a manager account? If so, we need to make sure to read the address of the "acting as" user. Right now, that only exists as a concept in client/common. You can either read the current user (not the current web3user), or use the `getAddresses` function from `authService`.